torch/_dynamo/eval_frame.py

```diff
@@ -339,11 +340,20 @@ def __init__(self):
         super().__init__(callback=None)
 
 
+def first_real_inst_idx(code):
+    if sys.version_info < (3, 11):
+        return 0
+    for inst in dis.get_instructions(code):
+        if inst.opname == "RESUME":
+            return inst.offset // 2
+
```

Comment:
Should this function return something if we reach the end of the loop?

Reply:
Should error since all code in 3.11 should have a RESUME.